inspection/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
It's annoying that this is picking up inspection stuff as well. I think merging master into this branch to get rid of that.